csharp/ql/integration-tests/all-platforms/autobuild/autobuild.csproj

```diff
@@ -3,7 +3,6 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>net5.0</TargetFramework>
```

Comment:
Maybe target `net9.0` as well.

Reply:
I tested it, and upgrading this will require a coordinated PR in `semmle-code` and `codeql`. In the interest of time, reducing noise and wrapping up ubuntu migration, I'd like to leave the upgrade to `net9` off the scope of this PR , and deal with it separatedly